roles/cdi/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
+---
 cdi_namespace: "{{ cdi_image_namespace | default('kube-system') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
```

Comment:
```cdi_release_tag``` should be ```latest```.  We'll test with latest until there's a new tagged image.

Reply:
done